src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../types';
+import { findPersonByName } from '../utils/findPersonByName';
+
+interface Props {
+  person: Person;
+  people: Person[];
+}
+
+export const PersonInfo: FC<Props> = ({ person, people }) => {
+  const { personSlug } = useParams();
+
+  const mother = findPersonByName(people, person.motherName);
+  const father = findPersonByName(people, person.fatherName);
```

Comment:
You are executing `findPersonByName` twice for each person. Consider optimizing it by creating a map of people by name before rendering the list. This way, you can access each person's data in constant time.